classes/PodsMeta.php

```diff
@@ -193,6 +194,7 @@ public function core() {
 					if ( ! pods_tableless() ) {
 						add_filter( 'add_term_metadata', array( $this, 'add_term_meta' ), 10, 5 );
 						add_filter( 'update_term_metadata', array( $this, 'update_term_meta' ), 10, 5 );
+						add_filter( 'update_term_metadata_by_id', array( $this, 'update_term_meta_by_id' ), 10, 4 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: c79682639f6e8310406e7c84652ce115</sub>